### PR TITLE
feat: automatic loading of `.env` file

### PR DESCRIPTION
**Pull request by beyazit**
added dotenv option to read environment variables files

### CODE REVIEW

File: src/dev/imports.ts [hunk 1]
```diff
@@ -20,3 +20,8 @@ export function twindImports(imports: Record<string, string>) {
   imports["twind"] = `https://esm.sh/twind@${RECOMMENDED_TWIND_VERSION}`;
   imports["twind/"] = `https://esm.sh/twind@${RECOMMENDED_TWIND_VERSION}/`;
 }
+
+export function dotenvImports(imports: Record<string, string>) {
+  // always use latest version
+  imports["$std/"] = `https://deno.land/std/`;
```
Comment (lucacasonato) [inline src/dev/imports.ts]: This should be a versioned import.

**Review by lucacasonato — COMMENTED** [review]
I like this. Let's actually enable dotenv always, with no opt-out in the init script. We should try to keep the options in the init script nice and simple so users don't have to make a bunch of decisions when they are initially setting up their Fresh project :)

**Comment by thorwebdev** [comment]
Just found this while looking for docs for `dotenv`. Wondering what the status here is? Is it worthwhile to contribute some docs or is `dotenv` default landing soon?

File: src/dev/imports.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ export const RECOMMENDED_PREACT_RTS_VERSION = "5.2.3";
 export const RECOMMENDED_PREACT_SIGNALS_VERSION = "1.0.3";
 export const RECOMMENDED_PREACT_SIGNALS_CORE_VERSION = "1.0.1";
 export const RECOMMENDED_TWIND_VERSION = "0.16.17";
+export const RECOMMENDED_STD_VERSION = "0.155.0";
```
Comment (iuioiua) [inline src/dev/imports.ts]:
```suggestion
export const RECOMMENDED_STD_VERSION = "0.187.0";
```
This is the current latest version.

**Review by iuioiua — CHANGES_REQUESTED** [review]
Could you please remove any opt-in/opt-out options, as Luca requested? I'll merge once, the conflict and my suggestion are also addressed 👍🏾

**Review by iuioiua — APPROVED** [review]
I've added a small sentence pointing out this new behaviour in the docs. Thanks for this, @0bl!

**Pull request merged**